README.md

```diff
@@ -2,36 +2,20 @@
 
 Capacitor plugin for HyperTrack generation SDKs
 
-## Install
+## Build plugin (local machine)
 
 ```bash
-npm install hypertrack-capacitor-plugin
-npx cap sync
+clone the repo git@github.com:hypertrack/sdk-ionic-capacitor.git
```

Comment:
have you tested this on clean machine? last time I tried to import, I had to install some global npm dependencies as well

please include all requirements like installed npm in the steps

Reply:
okay